app/src/config/constants/sub/features.js

```diff
@@ -106,5 +106,6 @@ FEATURES.LOCAL_FIRST_DESKTOP_APP = "local_first_desktop_app";
 FEATURES.LOCAL_WORKSPACE_COMPATIBILITY = "local_workspace_compatibility";
 
 FEATURES.API_CLIENT_DESCRIPTION_COMPATIBILITY = "api_client_description_compatibility";
+FEATURES.API_CLIENT_TYPE_COMPATIBILITY = "api_client_type_compatibility";
```

Comment:
update flag name. just `type` seems vague and might conflict with other flags we may want to add later on